### ISSUE TITLE
Add real-time slack

### ISSUE THREAD

**Issue by caseyyee** [MozVR/mozvr-site#163]
https://github.com/rauchg/slackin#realtime-badge

**Comment by ngokevin** [comment]
Nice, I need to do this for A-Frame site as well.

**Comment by cvan** [comment]
Yeah, I got this set up for https://github.com/cvan/slacknow. Will give y'all the links soon.
